app.ts

```diff
@@ -1,11 +1,15 @@
 import * as express from "express"
 import * as cors from "cors"
+import {globalErrorHandler} from './utils/globalErrorHandler';
```

Comment:
cudzysłowia maja byc jednolite. Alo piszemy " albo '

Reply:
OK, to idę w '.